packages/yq-4/packaging

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -exu
```

Comment:
Adding more packages never feels great, because it's more things to worry about carrying around and updating, but it does clean up some of the ugly transformations done via bash.

One thing that we should probably do is pick a different name for the package though since it's used at runtime. Bosh doesn't namespace packages, so if another release has a `yq-4` package, it would conflict and bosh wouldn't let you deploy the two releases on the same VM.

So for packages that are based on a 3rd party tool like this, you normally want to name them `postgres-yq-4` or something to ensure they don't conflict.

Reply:
Done